src/components/atoms/button/button.scss

```
@@ -0,0 +1,35 @@
+@use "../../variables" as *;
+
+.button {
+  background-color: $black;
+  text-decoration: none;
+  transition: all 0.2s ease;
+  font-weight: 400;
+  font-size: $fontsize-s;
+  color: $white;
+  width: 200px;
+  height: 55px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:hover {
+    color: $white;
+    background: $orange_bistre;
+  }
+
+  @media screen and(max-width:$breakpoint-lg) {
+    width: 200px;
+    height: 55px;
+  }
+
+  @media screen and(max-width:$breakpoint-md) {
+    width: 150px;
+    height: 55px;
+  }
+
+  @media screen and(max-width:$breakpoint-sm) {
+    weight: 180px;
```

```suggestion
    width: 180px;
```
weight -> width
お願いします！